rules/build_test.bzl

```diff
@@ -75,15 +99,13 @@ def build_test(name, targets, **kwargs):
             outs = [full_name + ".out"],
             testonly = 1,
             visibility = ["//visibility:private"],
-            # TODO: Does this need something else for Windows?
             cmd = "touch $@",
+            cmd_bat = "type nul > $@",
             **genrule_args
         )
 
-    native.sh_test(
+    _empty_test(
         name = name,
-        # TODO: Does this need something else for Windows?
-        srcs = ["@bazel_skylib//rules:empty_test.sh"],
         data = test_data,
         size = kwargs.pop("size", "small"),  # Default to small for test size
```

Comment:
```suggestion
        size = kwargs.pop("size", "small"),  # Default to small for test size
        is_windows = select({
                "@bazel_tools//src/conditions:host_windows": True,
                "//conditions:default": False,
        }),
```

Reply:
This is evaluatited in the `BUILD`, file, therefore the select is resolved as expected.